src/ClientRuntime/Microsoft.Rest.ClientRuntime.Azure.Authentication/Properties/AssemblyInfo.cs

```diff
@@ -9,8 +9,8 @@
 
 [assembly: AssemblyTitle("Microsoft Rest Azure Client Runtime Authentication")]
 [assembly: AssemblyDescription("Client authentication infrastructure for Azure client libraries.")]
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.2.10.0")]
```

Comment:
Did you mean 2.2.11? if so please keep AssemlbyVersion as 2.0.0 since Patch update means to be non-breaking